### DISCUSSION TITLE
Raw query failed. Code when I try to add like in query

### DISCUSSION THREAD

**Discussion by anehkumar** [prisma/prisma#13175] [Q&A]
Hi, when I try to run prisma.$queryRaw 

```
await prisma.$queryRaw(
                Prisma.sql`SELECT products.id, products.name, company_products.description, company_products.media, company_products.brand_name, companies.name as company_name, value_chains.name as value_chain_name, industrial_sectors.name as sector_name 
                FROM products LEFT JOIN company_products on company_products.product_id=products.id LEFT JOIN companies on companies.id=company_products.company_id LEFT JOIN value_chains on value_chains.id=products.value_chain_id LEFT JOIN industrial_sectors on industrial_sectors.id=products.industrial_sector_id
                where companies.status=${status} and (products.name like '%${search}')
                GROUP BY products.id;`,
            )
```
I am getting this error

Invalid `prisma.queryRaw()` invocation:

  Raw query failed. Code: `N/A`. Message: `N/A`

**Comment by nurul3101** [comment]
Hey @anehkumar 👋 
Which database and `@prisma/client` version are you using? 

Depending upon the database you might need to add schema name before referencing the table.

**Reply by vladi-stevanovic** [reply]
👋 Hello @anehkumar  I'm going through pending GitHub Discussions to check if you need any help. Is this issue still open?
Please let us know if we can investigate it further or if you've already solved it, we'd love to know how!

It would be super useful for other users facing this problem! 🙌

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
